pkg/service/identity.go

```diff
@@ -32,8 +38,18 @@ func (i *IdentityService) GetPluginCapabilities(context.Context, *csi.GetPluginC
 	}, nil
 }
 
-//Probe
+// Probe checks the state of the connection to ovirt-engine
 func (i *IdentityService) Probe(ctx context.Context, request *csi.ProbeRequest) (*csi.ProbeResponse, error) {
-	klog.V(4).Infof("probe called with args: %#v", request)
-	return &csi.ProbeResponse{}, nil
+	c, err := i.ovirtClient.GetConnection()
+	if err != nil {
+		klog.Errorf("Could not get connection %v", err)
+		return nil, status.Error(codes.FailedPrecondition, "Could not get connection to ovirt-engine")
+	}
+
+	if err := c.Test(); err != nil {
+		klog.Errorf("Connection test failed %v", err)
+		return nil, status.Error(codes.FailedPrecondition, "Could not get connection to ovirt-engine")
```

Comment:
whould we return the underlying `err` as well in the status.Error so it will be visible? or the err should be kept only in the log?
Maybe add to the text message the err message?

Reply:
I'll add it

Reply:
that's could go in a new PR